plugins/org.komodo.importer/src/org/komodo/importer/ddl/DdlImporter.java

```diff
@@ -86,63 +88,67 @@ protected KomodoObject executeImport(UnitOfWork transaction, String content,
     }
 
     /**
-     * Perform the model import using the specified DDL File.  The DDL constructs must be valid to put directly beneath a model.
+	 * @throws KException  
+	 */
+    @Override
+    protected boolean handleExistingNode(UnitOfWork transaction,
+    		ImportOptions importOptions,
+    		ImportMessages importMessages) throws KException {
+
```

Comment:
Why does this only return true rather than concern itself with whether the parent exists or not. If the parent already exists then it may well have ddl text on its property hence already have sequenced children which may need to be removed first before sequencing can proceed.